package.json

```diff
@@ -20,7 +20,7 @@
     "storybook-docs": "start-storybook --docs",
     "build-storybook": "build-storybook -c .storybook -o dist/storybook",
     "test": "jest --watchAll",
-    "test-once": "jest",
+    "test-once": "jest -w 1",
```

Comment:
nice why not --maxWorkers=50%?

Reply:
@symphony-adnane I found it in several links. (For example: https://github.com/zcorky/moment/issues/26, https://discuss.circleci.com/t/jest-out-of-memory-error/32263)
So, I propose to try with '-w 1' (This will prevent jest from accidentally spawning too many workers.).
And if the problem still occurs, then we'll try using 50%